src/ensembl/src/shared/types/thoas/cdna.ts

```diff
@@ -14,9 +14,11 @@
  * limitations under the License.
  */
 
+import { Sequence } from './sequence';
+
 export type CDNA = {
   start: number;
   end: number;
   length: number;
-  sequence_checksum?: string;
+  sequence?: Sequence;
```

Comment:
Given how we treat the rest of the fields in this type, and in the CDS type, `sequence` should be declared as a mandatory, not an optional field.

Both here and in the CDS type file.

Reply:
Done